Applications/Games/Warcraft III Expansion Set/Online/script.js

```diff
@@ -1,13 +1,14 @@
 include(["Engines", "Wine", "QuickScript", "OnlineInstallerScript"]);
-include(["Engines", "Wine", "Verbs", "corefonts"]);
 include(["Engines", "Wine", "Verbs", "vcrun2015"]);
+include(["Engines", "Wine", "Verbs", "corefonts"]);
 
 new OnlineInstallerScript()
-    .name("Warcraft III")
+    .name("Warcraft III Expansion Set")
     .editor("Blizzard")
-    .applicationHomepage("http://eu.blizzard.com/en-gb/games/war3/")
-    .author("FalseCAM")
-    .url("https://www.battle.net/download/getLegacy?product=WAR3&locale=en-US&os=WIN")
+    .applicationHomepage("https://eu.battle.net/shop/en/product/warcraft-iii-the-frozen-throne")
```

Comment:
"http://eu.blizzard.com/en-gb/games/war3/" seems to be the official Blizzard homepage for WC3 so I would prefer that instead of the shop page of TFT.

Reply:
Well, it seems to be the official homepage for WC3 RoC (they don't seem to have one for TFT for some reason). I've changed it to that page as you suggested.